test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/UWPRestoreTests.cs

```diff
@@ -330,7 +330,10 @@ public async Task UWPRestore_BlankUWPApp()
                 Assert.Equal(0, logger.Warnings);
                 Assert.Equal(118, result.GetAllInstalled().Count);
 
-                Assert.Equal(expectedJson.ToString(), lockFileJson.ToString());
+                // TODO: skipping comparing assets file since NuGet.org is doing repo sign for all the existing
```

Comment:
TODO => tracking issue, even if it's the same one.

Reply:
this is not related to this change per se instead failing on dev or 4.9.0 branches as well. Just wanted to fix the build. tracking issue# https://github.com/NuGet/Home/issues/7361